pkg/subnet/upgrades/upgrade_file.go

```diff
@@ -13,33 +13,26 @@ import (
 	"github.com/ava-labs/avalanchego/utils/storage"
 )
 
-func WriteUpgradeFile(jsonBytes []byte, subnetName, upgradeFilesDir string) error {
+func WriteUpgradeFile(jsonBytes []byte, subnetName, subnetsRoot string) error {
 	var (
 		exists bool
 		err    error
 	)
 
-	subnetPath := filepath.Join(upgradeFilesDir, subnetName)
-	updateBytesFileName := filepath.Join(subnetPath, constants.UpdateBytesFileName)
+	subnetDir := filepath.Join(subnetsRoot, subnetName)
```

Comment:
This should be done in the application package

Reply:
I wanted explicitly to avoid dragging the `app` reference in here as it currently doesn't need it.

Reply:
OK moved it to app finally